.github/workflows/ci.yml

```
@@ -10,7 +10,7 @@ on:
       - created
 
 env:
-  ESM_NAME: 'openmrs-esm-form-engine-lib'
+  ESM_NAME: '@openmrs/esm-form-engine-lib'
   JS_NAME: 'openmrs-esm-form-engine-lib.js'
```

```suggestion
  ESM_NAME: '@openmrs/esm-form-engine-lib'
  JS_NAME: 'openmrs-esm-form-engine-lib.js'
  TURBO_API: 'http://127.0.0.1:9080'
  TURBO_TOKEN: ${{ secrets.TURBO_SERVER_TOKEN }} 
  TURBO_TEAM: ${{ github.repository_owner }}
  
```

Can we move the `TURBO` env vars here too and remove the nested `env` key in the `build` job?